plugin/trino-hive/src/test/java/io/trino/plugin/hive/TestOrcPageSourceMemoryTracking.java

```diff
@@ -467,7 +467,7 @@ public void testScanFilterAndProjectOperator()
 
             // memory usage varies depending on stripe alignment
             long memoryUsage = driverContext.getMemoryUsage();
-            assertTrue(memoryUsage < 1000 || (memoryUsage > 350_000L && memoryUsage < 465_000L));
+            assertTrue(memoryUsage < 1000 || (memoryUsage > 150_000L && memoryUsage < 630_000L));
```

Comment:
* when this fails, the exception message will be "expected true, but was false".
it would be nice if the message included the value of memoryUsage.

* also, why we expected memory to be either `[0, 1000)` or `(150K, 630K)`?
where do the border values come from?

* nit: `L` is redundant

Reply:
> also, why we expected memory to be either [0, 1000) or (150K, 630K)?
where do the border values come from?

I don't know. They are arbitrary. Also test mentions 
```
        // Numbers used in assertions in this test may change when implementation is modified,
        // feel free to change them if they break in the future
```
so I changed them. In actual benchmarks memory usage actually dropped